stable/kommander/templates/ingress.yaml

```diff
@@ -13,7 +13,7 @@ metadata:
     traefik.frontend.rule.type: PathPrefixStrip
     ingress.kubernetes.io/auth-secret: ops-portal-htpasswd
     ingress.kubernetes.io/auth-type: basic
-    ingress.kubernetes.io/auth-header-field: X-WebAuth-User
+    ingress.kubernetes.io/auth-header-field: X-Forwarded-User
```

Comment:
I don't think this will work if the `auth-type` is still `basic`.

Reply:
actually, scratch that. This is `auth-header-field` not `auth-responses-header`.